src/app/graphql.service.ts

```diff
@@ -1,11 +1,11 @@
-import { gql } from 'apollo-angular';
+// Angular
```

Comment:
The changes in our GQL service are compiled automatically based on our queries and schema. We don't modify these file manually.

Reply:
Good to know, thanks!